packages/rum/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -356,6 +356,46 @@ describe('rum track loading type', () => {
   })
 })
 
+describe('rum track view is active', () => {
+  let setupBuilder: TestSetupBuilder
+  let handler: jasmine.Spy
+  let getViewEvent: (index: number) => View
+
+  beforeEach(() => {
+    ;({ handler, getViewEvent } = spyOnViews())
+
+    setupBuilder = setup()
+      .withFakeClock()
+      .withFakeLocation('/foo')
+      .beforeBuild(({ location, lifeCycle }) => {
+        lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
+        return trackViews(location, lifeCycle)
+      })
+  })
+
+  afterEach(() => {
+    setupBuilder.cleanup()
+  })
+
+  it('should collect initial view as active', () => {
+    setupBuilder.build()
+    expect(getViewEvent(0).isActive).toBe(true)
+  })
+
+  it('should collect old view as inactive and new one as active after a route change', () => {
+    setupBuilder.build()
+    history.pushState({}, '', '/bar')
+    expect(getViewEvent(1).isActive).toBe(false)
+    expect(getViewEvent(2).isActive).toBe(true)
+  })
+
+  it('should not collect view as inactive after any location change', () => {
```

Comment:
```suggestion
  it('should keep view as active after a search change', () => {
```
FMU, any location change includes route change